### DISCUSSION TITLE
Welcome to TOCropViewController Discussions!

### DISCUSSION THREAD

**Discussion by TimOliver** [TimOliver/TOCropViewController#452] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MichaelMeinberg** [comment]
Hey Tim, I love your crop VC, it really works perfectly for my app - well done.  The example works well for me, but in my app I just can't seem to get it to recognize the code, it gives me an "Cannot find.." error when trying to add the CropViewControllerDelegate and call CropViewController.

I tried adding it to Cocopod, doing a POD INSTALL, and I tried the Swift Package too, neither seems to work.  It shows the Swift Package TOCropViewController 2.5.5, but I can't call it.   I added a bridging header and added #import "TOCropViewController.h" to it, no luck. 

Not sure why this is happening, any ideas?  

Thanks.

**Comment by vipinsainitech** [comment]
"toolbarPosition top" is overlap with title and statusBar in iPhone6s (without notch iPhones) with present mode

First Thank you so much for creating this type library for us.

When we set toolbarPosition top and open cropController with present then facing this issue. If we first dismiss and then push it's working fine but other issue like save or cancel not work proper or if we push without dismiss then on cancel popTo camera view and retake and save button not work.

Can you please resolve this on urgent basic? It's help me a lots.


![IMG_7657](https://user-images.githubusercontent.com/16730481/106867179-4d021400-66f3-11eb-88cc-2ac6110cb068.PNG)

**Comment by GoDo-2202** [comment]
How to change the arrow shape edges crop to a circle? Please help me

**Comment by elijah-rappaport** [comment]
Is there an option anywhere to let the user choose the size of the final image, just like the iOS Mail app does? It presents a message saying "This message is 6.5MB. You can reduce message size by scaling the image to one of the sizes below."
Small (196 KB)
Medium (717 KB)
Large (2.4 MB)
Actual Size (6.5 MB)
